.github/workflows/update-mlflow.yml

```diff
@@ -0,0 +1,35 @@
+name: Update mlflow tags
+on:
+  schedule:
+  - cron: 0 0 * * *
+env:
+  PIP_EXTRA_INDEX_URL: ${{ format('https://{0}:{1}@{2}/{0}/{3}', secrets.DEVPI_USER,
+    secrets.DEVPI_PASS, secrets.DEVPI_HOST, secrets.DEVPI_INDEX) }}
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: make setup
+    - name: Run update mlflow script
+      run: python update_action.py --input “.neuro/*.y*ml" --name mlflow
+    - id: generate_token
+      uses: tibdex/github-app-token@v1
+      with:
+        app_id: ${{ secrets.BOT_APP_ID }}
+        private_key: ${{ secrets.BOT_PRIVATE_KEY }}
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v3.10.1
+      with:
+        token: ${{ steps.generate_token.outputs.token }}
+        branch: update/image-tags
+        title: Auto-update image tags
+        commit-message: Auto-update image tags
+        body: |
+          Update versions of image in helm values
+          configs to latest version
```

Comment:
I'm not sure this content is correct. There is no helm, neither images.
Lets add relevant info:
- simple way: just state that we are updating action versions.
- fancy way: we could perform `::set-output::` in the Python script below and [use it](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idoutputs) in PR body.

Reply:
@dmitrynikolayev , this issue is still actual. 
If you're not agreeing with the suggestion - let's discuss it here.

Reply:
Hint: https://stackoverflow.com/a/59201610